addons/core/XEH_postInit.sqf

```diff
@@ -2,75 +2,27 @@
-		["2BNB Flecktarn - Medic",[["BWA3_G36A3","","","BWA3_optic_CompM2",["BWA3_30Rnd_556x45_G36",30],[],""],[],["BWA3_P8","","","",["BWA3_15Rnd_9x19_P8",15],[],""],["BWA3_Uniform2_sleeves_Fleck",[["BWA3_Beret_PzGren",1],["ACE_elasticBandage",5],["ACE_packingBandage",10],["ACE_adenosine",1],["ACE_CableTie",2],["ACE_EarPlugs",1],["ACE_epinephrine",2],["ACE_IR_Strobe_Item",1],["ACE_morphine",2],["ACE_tourniquet",4],["ACE_MapTools",1],["ACE_Flashlight_XL50",1]]],["BWA3_Vest_Medic_Fleck",[["ACE_surgicalKit",1],["BWA3_30Rnd_556x45_G36",10,30],["SmokeShell",4,1],["HandGrenade",2,1],["SmokeShellGreen",1,1],["SmokeShellBlue",1,1]]],["BWA3_TacticalPack_Fleck",[["ACE_EntrenchingTool",1],["ACE_elasticBandage",40],["ACE_bloodIV_250",14],["ACE_bloodIV_500",6],["ACE_adenosine",5],["ACE_epinephrine",10],["ACE_morphine",10],["ACE_tourniquet",2],["ACE_packingBandage",10]]],"BWA3_M92_Fleck","",[],["ItemMap","ItemGPS","TFAR_anprc152","ItemCompass","ACE_Altimeter",""]]] call ace_arsenal_fnc_addDefaultLoadout;
-		["2BNB Flecktarn - Pilot",[[],[],["BWA3_MP7","","","",["BWA3_40Rnd_46x30_MP7",40],[],""],["BWA3_Uniform_Crew_Fleck",[["BWA3_Beret_HFlieger",1],["ACE_elasticBandage",5],["ACE_packingBandage",10],["ACE_adenosine",1],["ACE_CableTie",2],["ACE_epinephrine",2],["ACE_IR_Strobe_Item",1],["ACE_morphine",2],["ACE_tourniquet",4],["ACE_MapTools",1],["ACE_Flashlight_XL50",1],["ACE_EarPlugs",1],["BWA3_40Rnd_46x30_MP7",2,40],["SmokeShellOrange",1,1],["SmokeShellGreen",1,1]]],[],[],"BWA3_Knighthelm","",[],["ItemMap","ItemGPS","TFAR_anprc152","ItemCompass","ACE_Altimeter",""]]] call ace_arsenal_fnc_addDefaultLoadout;
-		["2BNB Flecktarn - Rifleman",[["BWA3_G36A3","","","BWA3_optic_CompM2",["BWA3_30Rnd_556x45_G36",30],[],""],[],["BWA3_P8","","","",["BWA3_15Rnd_9x19_P8",15],[],""],["BWA3_Uniform2_sleeves_Fleck",[["BWA3_Beret_PzGren",1],["ACE_elasticBandage",5],["ACE_packingBandage",10],["ACE_adenosine",1],["ACE_CableTie",2],["ACE_EarPlugs",1],["ACE_epinephrine",2],["ACE_IR_Strobe_Item",1],["ACE_morphine",2],["ACE_tourniquet",4],["ACE_MapTools",1],["ACE_Flashlight_XL50",1]]],["BWA3_Vest_Rifleman_Fleck",[["BWA3_30Rnd_556x45_G36",12,30],["SmokeShell",4,1],["HandGrenade",2,1],["SmokeShellGreen",1,1],["SmokeShellBlue",1,1]]],["BWA3_TacticalPack_Fleck",[["ACE_EntrenchingTool",1]]],"BWA3_M92_Fleck","",[],["ItemMap","ItemGPS","TFAR_anprc152","ItemCompass","ACE_Altimeter",""]]] call ace_arsenal_fnc_addDefaultLoadout;
-		//-------------------------------------------------------------------------------
+	BNB_LoadoutUnits = [];
+	publicVariable "BNB_LoadoutUnits";
```

Comment:
You don't need to broadcast an empty variable just yet. Wait until you have a the data in it. I explain in the next comment why